piqe_ocp_lib/tests/resources/test_ocp_configs.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import pytest
+from piqe_ocp_lib.api.resources.ocp_configs import OcpConfig
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+# Note : This unittest will use image registry config. if you want to run against any other config,
+# Please change the API_VERSION
+KIND = "Config"
+API_VERSION = "imageregistry.operator.openshift.io/v1"
+pytest.config_response = None
+pytest.config_response_before_update = None
+
+
+@pytest.fixture(scope="session")
```

Comment:
Please provide PyDoc for the test methods.

Reply:
Done